src/index.js

```diff
@@ -0,0 +1,20 @@
+import _ from 'lodash'; // eslint-disable-line
```

Comment:
- [ ] Please don't disable the linter rules, they are here to help you improve your code readability.